DuggaSys/diagram.php

```diff
@@ -80,6 +80,15 @@
                 <img src="../Shared/icons/diagram_zoomout.svg"/>
             </div>
         </fieldset>
+        <fieldset>
+        <legend>Toggle</legend>
+        <div id="gridToggle" class="diagramIcons" onclick='toggleGrid()'>
+                    <img src="../Shared/icons/diagram_grid.svg"/>
+                </div>
+                <div id="rulerToggle" class="diagramIcons active" onclick='toggleRuler()'>
+                    <img src="../Shared/icons/diagram_ruler.svg"/>
+                </div>
+        </fieldset>
```

Comment:
Indentation isn't really the most beautiful. Please fix this. Apart from this everything is OK.

Reply:
Fixed